book/listings/gobject_properties/3/custom_button/mod.rs

```diff
@@ -0,0 +1,23 @@
+mod imp;
+
+use glib::{BindingFlags, Object};
+use gtk::{glib, Align, Orientation};
+use gtk::{prelude::*, BoxBuilder};
+use gtk::{Application, ApplicationWindowBuilder};
+use std::cell::RefCell;
+
+glib::wrapper! {
+    pub struct CustomButton(ObjectSubclass<imp::CustomButton>)
+        @extends gtk::Button, gtk::Widget;
```

Comment:
Any reason for putting those into a new line?

Reply:
Is there a reason not to?
I like it more, because if it is in the same line users with a smaller monitor might overlook it

Reply:
That's way below the characters per line limit set by rustfmt i think, no preferences though :-)

Reply:
I usually put a linebreak before the @extends / @implements part as the line tends to get quite long otherwise

Reply:
@sdroege you have tagged two random persons :p

Reply:
Well!